students/ann/loops-js-exercises/animated-chart/chart.js

```diff
@@ -0,0 +1,21 @@
+const number = parseInt(prompt('number'))
+const midLine = Math.floor(number / 2);
+
+for (let i = 0; i < midLine; i++) {
+    document.write(`<div class="container ${i}">`);
+
+    for (let j = i; j >= 0; j--) {
+        document.write(`<div class="bar green-bar ${j}" ></div>`);
+    }
+    document.write(`</div>`);
+}
+
+
+for (let i = midLine; i >= 0; i--) {
+    document.write(`<div class="container ${i}">`);
+
+    for (let j = i; j >= 0; j--) {
+        document.write(`<div class="bar red-bar ${j}"></div>`);
```

Comment:
when using document write, it creates a child element to the body, so if you want to create a container and a childe element insside container, you must build the string and after `for` statement call document.write